lib/client/clock-client.js

```diff
@@ -52,13 +69,13 @@ client.render = function render (xhr) {
   $('#bgnow').toggleClass('stale', (now - last > threshold));
 
   // Generate and insert the clock.
-  let timeDivisor = (client.settings.timeFormat) ? client.settings.timeFormat : 12;
+  let timeDivisor = parseInt(client.settings.timeFormat ? client.settings.timeFormat : 12, 10);
   let today = new Date()
     , h = today.getHours() % timeDivisor;
-  if (timeDivisor == 12) {
-    h = (h == 0) ? 12 : h; // In the case of 00:xx, change to 12:xx for 12h time
+  if (timeDivisor === 12) {
+    h = h || 12; // In the case of 00:xx, change to 12:xx for 12h time
```

Comment:
Formatting

Reply:
This is not only formatting. For h == 4 this will be changed to h==12. Please restore old behaviour, perhaps with === instead of ==

Reply:
fixed on my branch